p/psad/psad.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Tested on     : UBI 8.3
+# Maintainer    : Nishikant Thorat <nishikant_thorat@ibm.com>
+# NOTE          : This script helps to install psad on UBI 8.3, with psad.conf 
+#                 with additional parameters by test suite, which may or may not
+#                 be needed.
+#
+# ----------------------------------------------------------------------------
+function Build
+{
+	./psad_UBI_8.3.sh
+}
+
+function Install
+{
+	yum install -y git wget vim make gcc initscripts chkconfig iptables iproute net-tools ruby-devel gcc make rpm-build rubygems
+	yum install -y cpan perl gettext mailx m4
+	export PERL_MM_USE_DEFAULT=1; cpan Date::Calc Data::Dumper IPTables::Parse NetAddr::IP Unix::Syslog IPTables::ChainMgr
+	git clone https://github.com/rfc1036/whois; cd whois/;  make; make install; cd ..
+	#yum install -y sendmail-cf
+	# if sendmail-cf is not available then,
+	#
+	# Sendmail installation  - Start
+	#
+	wget https://ftp.icm.edu.pl/packages/sendmail/sendmail.8.15.1.tar.gz; tar -xvf sendmail.8.15.1.tar.gz; cd sendmail-8.15.1;
+
+	if [ ! -d /usr/man ]; then mkdir -p /usr/man;fi
+	if [ ! -d /usr/man/man1 ]; then mkdir -p /usr/man/man1; fi
+	if [ ! -d /usr/man/man5 ]; then mkdir -p /usr/man/man5; fi
+	if [ ! -d /usr/man/man8 ]; then mkdir -p /usr/man/man8; fi
+
+	groupadd -f smmsp;
+	useradd  smmsp -g smmsp;
+	sed  's/ldl/ldl -lresolv/' devtools/OS/Linux > devtools/OS/Linux.new; mv -f devtools/OS/Linux.new devtools/OS/Linux; ./Build -c;
+	cd sendmail;  ./Build install; cd ../..
+	#
+	# Sendmail installation  - End
+	#
+	mkdir -p /var/log/psad/tmp
+	#mkdir -p ./psad/test/psad-install/var/log/psad/tmp
+	# Install psad rpm
+	rpm -ihv psad-3.0-1.ppc64le.rpm
+	# Start psad
+	psad start
+}
+
+
+while getopts bich value
+do
+        case "$value" in
+                b) echo "Building Psad ..."
+			Build
+                        ;;
+                i) echo "Installing Psad ..."
+			Install
+                        ;;
+                c) echo "Building and installing Psad ..."
+			Build
+			Install
+                        ;;
+		h) echo "Usage: "
+                   echo "-b    Only builds psad."
+                   echo "-i    Only installs psad. Kindly ensure, you will have RPM for psad, before using this option."
+                   echo "-c    Build and install psad."
+                   echo "-h    Print usage."
```

Comment:
Please fix the indentation.

Reply:
>     1. Does `psad_rhel_8.3_install.sh` have a relevance anymore?

This file is renamed. 
 
>     1. If we're able to generate the expected rpm in correct format using `psad_RHEL_8.2.sh` which can be installed and executed properly, can we maybe add an install script for that one?

We had given install script to Bhargavi as build and install machines were not same and we were trying to install rpm generated from UBI 8.3 on RHEL 8.2. For RHEL 8.2, build and install box will remain same and I was able to install rpm using "rpm -ivh <pkg>" command and able to start psad without any issue on "129.40.81.88". (I think, we dont have any other  box with RHEL 8.2)

Also, have corrected indentation.

Reply:
New commit `124739d deleting install script` took care of #1.